src/Organizations/OrganizationForm/OrganizationBankingInfoForm/BankingInformationFieldArray/BankingInformationFieldArray.test.js

```diff
@@ -0,0 +1,39 @@
+import { MemoryRouter } from 'react-router-dom';
+
+import { render, screen } from '@folio/jest-config-stripes/testing-library/react';
+import stripesFinalForm from '@folio/stripes/final-form';
+
+import { BankingInformationFieldArray } from './BankingInformationFieldArray';
+
+jest.mock('@folio/stripes-acq-components', () => ({
+  ...jest.requireActual('@folio/stripes-acq-components'),
+  RepeatableFieldWithValidation: jest.fn(() => 'RepeatableFieldWithValidation'),
+}));
+
+const FormWrapper = stripesFinalForm({
+  keepDirtyOnReinitialize: true,
+  navigationCheck: true,
+  subscription: { values: true },
+  validateOnBlur: true,
+})(({ children }) => <form>{children}</form>);
+
+const wrapper = ({ children }) => (
+  <MemoryRouter>
+    <FormWrapper onSubmit={jest.fn()}>
+      {children}
+    </FormWrapper>
+  </MemoryRouter>
+);
+
+const renderBankingInformationFieldArray = (props = {}) => render(
+  <BankingInformationFieldArray {...props} />,
+  { wrapper },
+);
+
+describe('OrganizationBankingInfoForm', () => {
+  it('should render RepeatableFieldWithValidation', async () => {
+    renderBankingInformationFieldArray();
+
+    expect(screen.getByText('RepeatableFieldWithValidation')).toBeInTheDocument();
```

Comment:
Please add some tests that events are handled correctly

Reply:
https://github.com/folio-org/ui-organizations/pull/580/files#diff-121f1c35023d3165551f59f421298d1e745f4bd62ef756617630012c557a4196R109-R123